<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormationControlClass</class>
 <widget class="QMainWindow" name="FormationControlClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1304</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FormationControl</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="CameraWidget" name="cameraWidget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
   </widget>
   <widget class="GLWindow" name="glwindow" native="true">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnApplyFormation">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>610</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>3. Apply formation</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lstFormations">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>500</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="currentRow">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>f(x) = 0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>f(x) = x</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>f(x) = |x|</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>f(x) = -0.5 x</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>f(x) = -|0.5 x|</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>f(x) = -|x|</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>f(x) = x^2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>f(x) = x^3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>f(x) = {sqrt(x),  x &gt;= 0 | -sqrt|x|, x &lt; 0}</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>f(x) = 0.05 sin(10 x)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnGo">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>500</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>2. GO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFindRobots">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>1. Find robots</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="stdoutput">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>490</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1304</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionNew_Terminal"/>
    <addaction name="actionOpen_ports"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionNew_Terminal">
   <property name="text">
    <string>New Terminal</string>
   </property>
  </action>
  <action name="actionOpen_ports">
   <property name="text">
    <string>Open ports</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CameraWidget</class>
   <extends>QWidget</extends>
   <header location="global">camerawidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLWindow</class>
   <extends>QWidget</extends>
   <header location="global">glwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="formationcontrol.qrc"/>
 </resources>
 <connections/>
</ui>
